README.md

```diff
@@ -7,9 +7,18 @@
 ## GemWallet: A crypto wallet & Web3 layer for the XRPL
 
 ![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen)
-![license](https://img.shields.io/badge/license-file%20LICENSE-blue)
+[![license](https://img.shields.io/badge/license-file%20LICENSE-blue)](https://github.com/GemWallet/gemwallet-extension/blob/feat/update-readme/LICENSE)
```

Comment:
The link seems incorrect, it point on the MR, it should be pointing on master

Reply:
good catch